packages/manager/apps/dedicated/client/app/components/cpanel-eol-banner/cpanel-eol-banner.component.js

```diff
@@ -0,0 +1,11 @@
+import template from './cpanel-eol-banner.html';
+import controller from './cpanel-eol-banner.controller';
+
+export default {
+  bindings: {
+    display: '<',
```

Comment:
**nitpick:** Unnecessary input imo. You can directly add `ng-if` in the template where this component is used.
```
<<cpanel-eol-banner data-ng-if="$ctrl.display"></cpanel-eol-banner>
```